docs/docusaurus/docs/core/set_up_a_gx_environment/_install_gx/_local_installation.md

```diff
@@ -0,0 +1,35 @@
+import ReleaseVersionBox from '../../../components/versions/_gx_version_code_box.mdx'
+
+GX 1.0 is a Python library and as such can be used with a local Python installation to access the functionality of GX through Python scripts.
+
+## Installation and setup
+
+1. (Optional) Activate your virtual environment.
+
+   If you created a virtual environment for your GX Python installation, navigate to the folder that contains your virtual environment and activate it by running the following command:
+
+   ```bash title="Terminal input"
+   source my_venv/bin/activate
+   ```
+
+2. Ensure you have the latest version of `pip`:
+
+   ```bash title="Terminal input"
+   python -m ensurepip --upgrade
+   ```
+
+3. Install the GX 1.0 library:
+
+   ```bash title="Terminal input"
+   pip install great_expectations
+   ```
+
+4. Verify that GX installed successfully with the CLI command:
```

Comment:
```suggestion
4. Verify that GX is installed successfully with the CLI command:
```
CLI command?

Reply:
Technically I think this is the last remnant of the CLI, but these references should be changed to 'terminal command' instead, since the rest of the CLI is gone.  I'll update it.